frontend/src/routes/servers/ChooseServer.svelte

```diff
@@ -140,7 +141,7 @@
     </form>
 </Modal>
 
-<div class="chooseServer">
+<section>
     <b>Your Servers</b><br><br>
```

Comment:
Get rid of every `<br>`, not just these which I mentioned, try performing entire repo search to find and delete every `<br>`